lib/router/app_router.dart

```diff
@@ -186,6 +190,19 @@ final eventBuyTicketsRoutes = AutoRoute(
   ],
 );
 
+final createEventRoutes = AutoRoute(
+  page: CreateEventRoute.page,
+  children: [
+    AutoRoute(
+      initial: true,
+      page: BaseCreateEventRoute.page,
```

Comment:
I think we can consolidate how we name for a feature 
ex: CreateEventBasicInfoPage, CreateEventSettingsPage, etc...

Reply:
then let's come with more generic pattern => Event + Domain + Setting 
ex: EventBasicInfoSetting, EventLocationSettings, EventTicketsSettings